core/sdk/src/api/session.rs

```diff
@@ -58,6 +60,7 @@ pub enum AccountSessionRequest {
     Get,
     Provision(ProvisionRequest),
     SetAccessToken(AccountSessionTokenRequest),
+    GetAccessToken,
```

Comment:
`Get` already gets AccessToken do we need another entry in the enumeration?

Reply:
But that does not have an expiry information. Only with that, we can invalidate the token in the ripple context. Since we don't have an event originating from thunder on token expiry.